src/main/kotlin/minesweeper/domain/MineSweeper.kt

```diff
@@ -1,22 +1,24 @@
 package minesweeper.domain
 
-data class MineSweeper(val mineCoordinateStrategy: MineCoordinateStrategy = RandomMineCoordinateStrategy) {
+class MineSweeper(
+    val boardFields: BoardFields,
+) {
+    val isEnd get() = boardFields.isAllOpenedNumberFields() || boardFields.isOpenedMineField()
+    val isWin get() = boardFields.isAllOpenedNumberFields()
```

Comment:
```suggestion
    val isEnd get() = isWin || boardFields.isOpenedMineField()
    val isWin get() = boardFields.isAllOpenedNumberFields()
```
끝난 상태는 이기거나 지뢰가 오픈된 경우라고 생각해보면 이렇게 표현해도 될 거 같아요!